U/Uno/build_tarballs.jl

```diff
@@ -0,0 +1,81 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "Uno"
+
+version = v"1.1.0"
+
+sources = [
+    GitSource(
+        "https://github.com/cvanaret/Uno.git",
+        "3bd224443a905c3d41b058a108db2b382a537ea4",
+    ),
+]
+
+script = raw"""
+cd $WORKSPACE/srcdir/Uno
+mkdir -p build
+cd build
+
+if [[ "${target}" == *mingw* ]]; then
+    LBT=libblastrampoline-5
+else
+    LBT=libblastrampoline
+fi
+
+cmake \
+    -DCMAKE_INSTALL_PREFIX=${prefix} \
+    -DCMAKE_TOOLCHAIN_FILE=${CMAKE_TARGET_TOOLCHAIN} \
+    -DCMAKE_BUILD_TYPE=Release \
+    -DBLA_VENDOR="libblastrampoline" \
+    -DMUMPS_INCLUDE_DIR=${includedir} \
+    -DMETIS_INCLUDE_DIR=${includedir} \
+    -DMUMPS_LIBRARY="${libdir}/libdmumps.${dlext}" \
+    -DMUMPS_COMMON_LIBRARY="${libdir}/libmumps_common.${dlext}" \
+    -DMUMPS_PORD_LIBRARY="${libdir}/libpord.${dlext}" \
+    -DMUMPS_MPISEQ_LIBRARY="${libdir}/libmpiseq.${dlext}" \
+    -DBLAS_LIBRARIES="${libdir}/${LBT}.${dlext}" \
+    -DLAPACK_LIBRARIES="${libdir}/${LBT}.${dlext}" \
+    ..
+
+make -j${nproc}
+cp libuno.a ${prefix}/lib/libuno.a
```

Comment:
Do you actually need to ship the static archive?  In any case, please always use verbose commands, they're always helpful in figuring out what actually happened, something like
```suggestion
install -Dvm 644 libuno.a -t ${prefix}/lib
```
but I'm not convinced we need the static library at all

Reply:
No, we don't need the static library, but it's the only `Product` that we can generate right now.  
I added it in the products so that we can see which platforms are still failing.

Uno doesn't support `make install`, so I copied the library by hand.  
@cvanaret It would be great to support `make install` so that CMake automatically installs the libraries, headers, and binaries in the prefix folder provided by the user with `-DCMAKE_INSTALL_PREFIX=${prefix}`.